docs/usage.rst

```diff
@@ -207,3 +207,47 @@ Subscriptions can be extended by using the ``CurrentSubscription.extend`` method
 .. warning::
 
     Subscription extensions are achieved by manipulating the ``trial_end`` of the subscription instance, which means that Stripe will change the status to ``trialing``.
+
+Customizing templates
+=====================
+
+Templates can be customized by copying one or more of them into the project's `TEMPLATES` folder, under a new folder `djstripe`, and editing accordingly. Frequently this folder is specified in the settings file as
+
+.. code-block:: python
+
+    TEMPLATES = [
+        {
+            'BACKEND': 'django.template.backends.django.DjangoTemplates',
+            'DIRS': [
+                join(BASE_DIR, 'templates'),  # <--- This folder
```

Comment:
`join` is a string method, the following will work better:

    '/'.join([BASE_DIR, '/templates/']),